app/controllers/admin/publishers/publisher_whitelist_updates_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::Publishers::PublisherWhitelistUpdatesController < Admin::PublishersController
+  def create
+    note = @publisher.notes.create!(note: params[:note], created_by_id: current_publisher.id)
+    PublisherWhitelistUpdate.create!(publisher: @publisher, enabled: params[:enable], publisher_note: note)
```

Comment:
```suggestion
    PublisherWhitelistUpdate.create!(publisher: @publisher, enabled: params[:enable].present?, publisher_note: note)
```

Reply:
This param is probably poorly named.  If it's `false` then we disable the whitelist, if it's `true` then we enable the whitelist. There is a non null constraint in the database, so this should always be set.

Reply:
`params[:enabled].present?` just handles the edge case of `params[:enabled]` being `nil` to be `false`

Reply:
I committed this and the test actually started failing.  Because what happens here is if we supply an explicit `false`, then `params[:enable].present?` will be `true` which isn't what we want